test/e2e-v2/script/env

```diff
@@ -23,7 +23,7 @@ SW_AGENT_CLIENT_JS_COMMIT=af0565a67d382b683c1dbd94c379b7080db61449
 SW_AGENT_CLIENT_JS_TEST_COMMIT=4f1eb1dcdbde3ec4a38534bf01dded4ab5d2f016
 SW_KUBERNETES_COMMIT_SHA=1335f15bf821a40a7cd71448fa805f0be265afcc
 SW_ROVER_COMMIT=0ae8f12d6eb6cc9fa125c603ee57d0b21fc8c6d0
-SW_BANYANDB_COMMIT=a044813184065a5bdb264ea2d453c70f4aac5711
+SW_BANYANDB_COMMIT=36584001aa3071eab89b9d811815979521a67a78
```

Comment:
[`a044813184065a5bdb264ea2d453c70f4aac5711` -> `36584001aa3071eab89b9d811815979521a67a78`](https://github.com/apache/skywalking-banyandb/compare/a044813184065a5bdb264ea2d453c70f4aac5711...36584001aa3071eab89b9d811815979521a67a78)

Some parts of these commits impact the feature.